### PR TITLE
DOCS-2953: Edit mlmodel service from QA

### PR DESCRIPTION
**Pull request by sguequierre**
* edits infer method, adds from robot, edits get resource name

### CODE REVIEW

File: static/include/services/apis/generated/mlmodel.md [hunk 1]
```diff
@@ -206,7 +226,7 @@ Get the `ResourceName` for this instance of the ML model service with the given
 **Example:**
 
 ```python {class="line-numbers linkable-line-numbers"}
-my_mlmodel_svc_name = MLModelClient.get_resource_name("my_mlmodel_svc")
+my_mlmodel_svc_name = my_mlmodel.get_resource_name("my_mlmodel_svc")
```
[the remark this answers is not in the document]
Comment (npentrel) [reply static/include/services/apis/generated/mlmodel.md:229]: why?

File: static/include/services/apis/generated/mlmodel.md [hunk 2]
```diff
@@ -46,7 +52,20 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+)
+
+myMLModel, err := mlmodel.FromRobot(machine, "mlmodel-1")
+
+input_tensors := ml.Tensors{
+    "image": tensor.New(
+        tensor.Of(tensor.Uint8),
+        tensor.WithShape(1, 384, 384, 3),
+        tensor.WithBacking(make([]uint8, 1*384*384*3)),
+  ),
```
Comment (npentrel) [inline static/include/services/apis/generated/mlmodel.md:67]: indentation is off

File: static/include/services/apis/generated/mlmodel.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply static/include/services/apis/generated/mlmodel.md:229]: oh you're right it's a class method, sorry for my misunderstanding. I did look into it originally and became confused thinking this was wrong but now that I look at the python sdk docs again it is at the same level as 'FromRobot', which makes sense as to why it should not be changed. editing ticket now and will fix this here

File: static/include/services/apis/generated/mlmodel.md [hunk 3]
```diff
@@ -46,7 +52,20 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+
+myMLModel, err := mlmodel.FromRobot(machine, "mlmodel-1")
+
+input_tensors := ml.Tensors{
+    "image": tensor.New(
+        tensor.Of(tensor.Uint8),
+        tensor.WithShape(1, 384, 384, 3),
+        tensor.WithBacking(make([]uint8, 1*384*384*3)),
+    ),
+}
```
Comment (sguequierre) [inline static/include/services/apis/generated/mlmodel.md:68]: @npentrel does this look better indentation-wise?

File: static/include/services/apis/generated/mlmodel.md [hunk 4]
```diff
@@ -46,7 +52,20 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 **Example:**
 
 ```go {class="line-numbers linkable-line-numbers"}
-input_tensors := ml.Tensors{"0": tensor.New(tensor.WithShape(1, 2, 3), tensor.WithBacking([]int{1, 2, 3, 4, 5, 6}))}
+import (
+    "go.viam.com/rdk/ml"
+    "gorgonia.org/tensor"
+)
+
+myMLModel, err := mlmodel.FromRobot(machine, "mlmodel-1")
```
Comment (npentrel) [inline static/include/services/apis/generated/mlmodel.md:60]:
```suggestion
myMLModel, err := mlmodel.FromRobot(machine, "my_mlmodel_service")
```

File: static/include/services/apis/generated/mlmodel.md [hunk 5]
```diff
@@ -97,6 +116,7 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 **Example:**
 
 ```go {class="line-numbers linkable-line-numbers"}
+myMLModel, err := mlmodel.FromRobot(machine, "mlmodel-1")
```
Comment (npentrel) [inline static/include/services/apis/generated/mlmodel.md:119]: We use the pattern "my_something" for everything, let's do the same here
```suggestion
myMLModel, err := mlmodel.FromRobot(machine, "my_mlmodel_service")
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**